src/Propagator/XTraceOptions/XTraceOptionsPropagator.php

```diff
@@ -32,21 +30,6 @@ public function fields(): array
 
     public function inject(mixed &$carrier, ?PropagationSetterInterface $setter = null, ?ContextInterface $context = null): void
     {
-        $setter ??= ArrayAccessGetterSetter::getInstance();
```

Comment:
Do we need `inject` function definition at all anymore?

Reply:
Yes, we need to implement all functions of TextMapPropagatorInterface, even it does nothing.

Reply:
Cool thanks!